### PR TITLE
Seperate image and video command support

### PR DESCRIPTION
**Pull request by FleshMobProductions**
Modularized !image functionality, so it works separately with images and videos now. command "image" will access file from the table "images" while command "video" will access files from the table "videos". 

Necessary: a new table with the name "videos" and the same table structure as "images"

### CODE REVIEW

File: scripts/db_controller.js [hunk 1]
```diff
@@ -161,92 +161,109 @@ module.exports = {
...
-            dbConnection.query(sqlInsertImage, function (error, result) {
-              if (error) throw error;
-                console.log("New image was added to images table.");
-            });
-            statusCallback(200);
-          }
-        });
-      }
-    });
+    InsertNewMedium(serverID, "images", authorID, title, url, statusCallback);
```
Comment (tomzig16) [inline scripts/db_controller.js]: Should be `InsertNewMedia`

Comment (tomzig16) [reply scripts/db_controller.js]: (same to the other places)

File: scripts/db_controller.js [hunk 2]
```diff
@@ -161,92 +161,109 @@ module.exports = {
...
-              }
-              else{
-                resultCallback(null);
-              }
-            });
-          }
-        });
-      }
-    });
+    GetMediumURL(serverID, "images", title, resultCallback);
```
Comment (tomzig16) [inline scripts/db_controller.js]: Should be `GetMediaURL`
(same for other places)

File: scripts/media_system.js [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+  }
+  // Print help for each command
+  if(args.length >= 2 && args[1] === "help"){
+    message.reply(Help(args[0], typeName, supportedFormats));
+    return;
+  }
+  // List all images available for the server
+  if(args[0] == "list"){
+    let getMediaFunction = isImage ? serverControl.GetAvailableImages : serverControl.GetAvailableVideos;
+    PrintInteractibleList(message, getMediaFunction);
```
Comment (tomzig16) [inline scripts/media_system.js]: `!video list` is missing

**Review by tomzig16 — CHANGES_REQUESTED** [review]
More to that, remove original `images_system` file as it is replaced with new one

**Comment by FleshMobProductions** [comment]
Implemented name convention changes, deleted image_system.js, make it visible that there are separate listings for image and video

**Pull request merged**